plenum/test/node_catchup/test_catchup_delayed_nodes.py

```diff
@@ -60,7 +75,8 @@ def testCatchupDelayedNodes(txnPoolNodeSet, nodeSetWithNodeAddedAfterSomeTxns,
     nodeX.stop()
     nodeY.stop()
     logger.debug("Sending requests")
-    sendReqsToNodesAndVerifySuffReplies(looper, wallet, client, 50)
+    sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle,
```

Comment:
pool refresh is not called before pool handle usage

fix it here and in all "add node tests" below